FTB/Signatures/CrashInfo.py

```diff
@@ -534,7 +534,8 @@ def __init__(self, stdout, stderr, configuration, crashData=None):
                                        |negative-size-param:\s*\(size=-\d+\)
                                        |not\smalloc\(\)-ed:     # Used in case of a wild free (on unallocated memory)
                                        |not\sowned:             # Used when calling __asan_get_allocated_size() on a pointer that isn't owned
-                                       |memcpy-param-overlap:\smemory\sranges\s\[) # Bad memcpy
+                                       |\w+-param-overlap:      # Bad memcpy/strcpy/strcat... etc
```

Comment:
Can you add tests for these other crashes? (or at least TODO if you don't have logs handy)

Reply:
I added a 'strcat-param-overlap' report to the test set. I don't think there are any other variations the require a test. From what I can find they all match "\w+-param-overlap:".